portfolio.html

```diff
@@ -0,0 +1,120 @@
+	</div>
+		<div class="textAboutMe">
+			<p>Hello my name is marcus and im a very dependable bloke you cna coinnt on me yes sir! SIKE get fucking Stick Bugged LOLjenagu nanrguao iraoubguoaerb nriopangpoeru nvrioaeungorae raueig rngioawngrpaiogn rnaignraeiogn rigoanoir rgao eiurbgo ragogaoie rng nrioagniroa ngraiognioraeng  rioangiaoerngrn gireang agnria ignarignro aignaioe nggnir ao;gnraiogngirona iong;raoi </p>
+	</div>
+	</section>
+		
+		<div class="portfolioTitle">
+			<h2>Portfolio</h2>
+		</div>
+		
+		<section class="threeBoxLayout">
+			<div class="leftBox">
+				<div class="leftImage">
+				<img src="https://images.pexels.com/photos/5137457/pexels-photo-5137457.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500" alt="Yo" class="portfolioImage">
+				</div>
+				<div class="leftText">
+				<a href="#" class="portfolioText">Left Upper Box hello this is a text test as im trying to figure sometihng out</a>
+				</div>
+			</div>
+			<div class="middleBox">
+				<div class="leftImage">
+				<img src="https://images.pexels.com/photos/5137457/pexels-photo-5137457.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500" alt="Yo" class="portfolioImage">
+				</div>
+				<div class="leftText">
+				<a href="#" class="portfolioText">Middle Upper Box hello this is a text test as im trying to figure sometihng out</a>
+				</div>
+			</div>
+			<div class="rightBox">
+				<div class="leftImage">
+				<img src="https://images.pexels.com/photos/5137457/pexels-photo-5137457.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500" alt="Yo" class="portfolioImage">
+				</div>
+				<div class="leftText">
+				<a href="#" class="portfolioText">Right Upper Box hello this is a text test as im trying to figure sometihng out</a>
+				</div>
+			</div>
+			<div class="leftBelowBox">
+				<div class="leftImage">
+				<img src="https://images.pexels.com/photos/5137457/pexels-photo-5137457.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500" alt="Yo" class="portfolioImage">
+				</div>
+				<div class="leftText">
+				<a href="#" class="portfolioText">Left Below Box hello this is a text test as im trying to figure sometihng out</a>
+				</div>
+			</div>
+			<div class="middleBelowBox">
+				<div class="leftImage">
+				<img src="https://images.pexels.com/photos/5137457/pexels-photo-5137457.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500" alt="Yo" class="portfolioImage">
+				</div>
+				<div class="leftText">
+				<a href="#" class="portfolioText">Left Box hellogggggg this is a text test as im trying to figure sometihng out</a>
+				</div>
+			</div>
+			<div class="rightBelowBox">
+				<div class="leftImage">
+				<img src="https://images.pexels.com/photos/5137457/pexels-photo-5137457.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500" alt="Yo" class="portfolioImage">
+				</div>
+				<div class="leftText">
+				<a href="#" class="portfolioText">Left Box hellogggggggg this is a text test as im trying to figure sometihng out</a>
+				</div>
+			</div>
+		</section>
+		
+	<footer  class="socialLinks">
```

Comment:
Can see that you're still working on this, but if you want any advice just let me know.

Reply:
Have added most of the changes you suggested bar the justify-content space between. Thank you for the help!